readinglist/views/article.py

```diff
@@ -67,6 +67,7 @@ class Options:
 @crud()
 class Article(BaseResource):
     mapping = ArticleSchema()
+    deleted_mark = ('status', 2)
```

Comment:
Actually I'd like to do both ``deleted: True`` and ``status: 2`` and if possible ask @rnewman to get rid of this ``status: 2`` thing since we cannot patch on it and it is less explicit than having ``deleted: True``